### PR TITLE
Move bfn asic drivers out of platform package

### PR DESCRIPTION
**Pull request by mkbalani**

### CODE REVIEW

File: platform/barefoot/bfn-modules/LICENSE [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Copyright (C) 2016  Microsoft, Inc
```
Comment (krambn) [inline platform/barefoot/bfn-modules/LICENSE]: Should we use this license?

File: platform/barefoot/rules.mk [hunk 2]
```diff
@@ -23,4 +24,4 @@ SONIC_ALL += $(SONIC_ONE_IMAGE) $(SONIC_ONE_ABOOT) \
 $(LIBSAIREDIS)_DEPENDS += $(BFN_SAI) $(WNC_OSW1800_PLATFORM) $(BFN_INGRASYS_PLATFORM) $(BFN_PLATFORM)  #$(LIBSAITHRIFT_DEV_BFN)
 
 # Runtime dependency on sai is set only for syncd
-$(SYNCD)_RDEPENDS += $(BFN_SAI) $(WNC_OSW1800_PLATFORM) $(BFN_INGRASYS_PLATFORM) $(BFN_PLATFORM)
+$(SYNCD)_RDEPENDS += $(BFN_MODULE) $(BFN_SAI) $(WNC_OSW1800_PLATFORM) $(BFN_INGRASYS_PLATFORM) $(BFN_PLATFORM)
```
Comment (krambn) [inline platform/barefoot/rules.mk:27]: Do we need this?

Comment (mkbalani) [reply platform/barefoot/rules.mk:27]: Isn't the presence of these modules mandatory at runtime for syncd to run?

File: platform/barefoot/bfn-modules/LICENSE (hunk 1 above)
Comment (mkbalani) [reply platform/barefoot/bfn-modules/LICENSE]: Not sure what to use here. Copied this over from license file from previously build bfn-platform-montara deb packaging

**Review by krambn — APPROVED** [review]

**Review by NStetskovych-zz — COMMENTED** [review]
Do we have separate package for asic drivers now?

**Comment by mkbalani** [comment]
@NStetskovych  yes, there is a single separate debian package which is used for all platforms

**Pull request merged**